lib/service/service.go

```diff
@@ -1084,10 +1084,17 @@ func initUploadHandler(ctx context.Context, auditConfig types.ClusterAuditConfig
 		}
 		return handler, nil
 	case teleport.SchemeS3:
-		config := s3sessions.Config{}
+		var fips *bool
+		if useFIPS {
+			fips = new(bool)
+			*fips = true
+		}
+		config := s3sessions.Config{UseFIPSEndpoint: fips}
```

Comment:
```suggestion
		config := s3sessions.Config{}
		if useFIPS {
			config.UseFIPSEndpoint = new(bool)
			*config.UseFIPSEndpoint = true
		}
```

This might read a little simpler. Up to you.

Reply:
I had it exactly like this but @rosstimothy suggested moving it into the initializer. I'll give it some thought.